src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -86,10 +86,14 @@ unsafe fn configure_llvm(sess: &Session) {
 // array, leading to crashes.
 
 const ARM_WHITELIST: &[(&str, Option<&str>)] = &[
+    ("aclass", Some("arm_target_feature")),
     ("mclass", Some("arm_target_feature")),
     ("rclass", Some("arm_target_feature")),
     ("dsp", Some("arm_target_feature")),
     ("neon", Some("arm_target_feature")),
+    ("v5te", Some("arm_target_feature")),
```

Comment:
IMO this shouldn't be needed because as this could be considered the base architecture. Anything older shouldn't need to be supported by rust, GCC removed pre-Armv5TE support earlier this year. So did Debian if I'm not mistaken too.

Reply:
`rustc` has a built-in `armv4t-unknown-linux-gnueabi` target that has the `v4t` feature enabled but the `v5te` feature disabled. I checked that stuff like `qadd` (IIRC? It was a DSP / SIMD32 instruction) compiles for `armv5te-unknown-linux-gnueabi` (another built-in target) but it doesn't for `armv4t-unknown-linux-gnueabi` (i.e. LLVM emits an error).